.netlify

```diff
@@ -0,0 +1 @@
+{"site_id":"d7bf54c3-d97a-44fe-a810-fdfdb42ace3d","path":"packages/site/dist"}
```

Comment:
I don't think it _needs_ to be secure. There is another generated file that lives in `$HOME/netlify` that contains an access token specific to your machine. This site id is just the pointer to our `http://<rando-hash>.stenographer-panda-45372.netlify.com` domain. Since only our team is going to deploy the site however, I think it'd be fine if this file is kept out of the repo. I'll trust your judgement on this one.